### ISSUE TITLE
EveryBeam/wsclean incompatibility in the singularity

### ISSUE THREAD

**Issue by botteon** [mhardcastle/ddf-pipeline#330]
Hi, I tried to build a singularity with the recipe in the repository, but it crashes while compiling wsclean:

```
......
-- Found EveryBeam 0.5.1.
--   EveryBeam include dir: /usr/local/include
--   EveryBeam lib: /usr/local/lib/libeverybeam.so
-- EveryBeam version is incompatible
CMake Error at CMakeLists.txt:180 (message):
  WSClean needs EveryBeam version 0.5.x with x>=2 - but found version 0.5.1


-- Configuring incomplete, errors occurred!
See also "/usr/local/src/wsclean/build/CMakeFiles/CMakeOutput.log".
See also "/usr/local/src/wsclean/build/CMakeFiles/CMakeError.log".
FATAL:   While performing build: while running engine: exit status 1
```

I guess that we stick to an old EveryBeam because of the usage of a specific DP3 version, right?

 https://github.com/mhardcastle/ddf-pipeline/blob/c42e8c682e5376a2e33da655b9bc16758ce75432/ddf-py3.singularity#L116

If not possible to move forward with DP3 (and thus with EveryBeam), the recipe should point to an old wsclean to have a working sigularity. The latest wsclean is not required to run facetselfcal on LoTSS data..

https://github.com/mhardcastle/ddf-pipeline/blob/c42e8c682e5376a2e33da655b9bc16758ce75432/ddf-py3.singularity#L147

**Comment by botteon** [comment]
Update: I tried to install the latest released version of wsclean (v3.3), but in this case EveryBeam v0.5.1 is too recent:

```
...
-- Found EveryBeam 0.5.1.
--   EveryBeam include dir: /usr/local/include
--   EveryBeam lib: /usr/local/lib/libeverybeam.so
-- EveryBeam version is incompatible
CMake Error at CMakeLists.txt:180 (message):
  WSClean needs EveryBeam version 0.4.x with x>=0 or 0.5.0 - but found
  version 0.5.1


-- Configuring incomplete, errors occurred!
See also "/usr/local/src/wsclean/build/CMakeFiles/CMakeOutput.log".
See also "/usr/local/src/wsclean/build/CMakeFiles/CMakeError.log".
```

Wsclean versions <=3.2 instead depend on EveryBeam <=0.4 (https://gitlab.com/aroffringa/wsclean/-/releases)

**Comment by mhardcastle** [comment]
Agreed that the current singularity recipe doesn't build any more. Cyril has a working version and I think we need to upgrade the one in the repo.
